File: Policie-FE/src/app/Models/GpsData.ts
export interface GpsData {
  type: string;
  dateTime: Date;
  latitude: number;
  longitude: number;
  accuracy: number;
  altitude: number | null;
  geoidHeight: number | null;
  speed: number | null;
  bearing: number | null;
  satUsed: number;
  satInView: number;
  name: string | null;
  description: string | null;
}

File: Policie-BE-2/Models/GpsData.cs
using System.ComponentModel.DataAnnotations;

namespace Policie_BE_2.Models
{
    public class GpsData
    {
        [Key]
        public int Id { get; set; }
        public string Type { get; set; }
        public DateTime DateTime { get; set; }
        public double Latitude { get; set; }
        public double Longitude { get; set; }
        public double Accuracy { get; set; }
        public double? Altitude { get; set; }
        public double? GeoidHeight { get; set; }
        public double? Speed { get; set; }
        public double? Bearing { get; set; }
        public int SatUsed { get; set; }
        public int SatInView { get; set; }
        public string? Name { get; set; }
        public string? Description { get; set; }
    }
}
